compio-runtime/src/runtime/mod.rs

```diff
@@ -25,17 +27,19 @@ use crate::{
     BufResult, Key,
 };
 
-pub(crate) struct Runtime {
+pub(crate) struct RuntimeInner {
+    id: Uuid,
```

Comment:
Why do we need uuid instead of a cheaper id generator (increment counter etc.)?

Reply:
Good catch. Maybe I should use an `AtomicUsize` instead.